### PR TITLE
fix: remove api-docs and swagger package

### LINKED ISSUE #602: Remove swagger docs
## Why? 
- [API contracts](https://github.com/Real-Dev-Squad/website-api-contracts) serves the same purpose.

## Remove swagger docs comments from all route files (Assigned to Rajat)
- Remove the comments
Eg - [Click here](https://github.com/Real-Dev-Squad/website-backend/blob/cc9476dec38137093f7108bcd2566b4bb5b99053/routes/auth.js#L5-L34)

## Remove `/api-docs`  (Assigned To Poonam)
- https://github.com/Real-Dev-Squad/website-backend/blob/develop/middlewares/index.js#L51-L58
- Remove swagger package https://github.com/Real-Dev-Squad/website-backend/blob/cc9476dec38137093f7108bcd2566b4bb5b99053/package.json#L38-L39

### PR DESCRIPTION
**Pull request by poonammohata**
resolves #602

### CODE REVIEW

**Comment by pallabez** [comment]
Can you fix the lint error? Also, check 
- https://github.com/Real-Dev-Squad/website-backend/pull/35
and remove things that were added there as well.

**Comment by poonammohata** [comment]
okh will do these changes

**Comment by poonammohata** [comment]
@pallabez fixed lint error, and removed things that were added in #35

File: package.json [hunk 1]
```diff
@@ -11,8 +11,7 @@
     "test-integration": "sh scripts/tests/testIntegration.sh",
     "test-unit": "sh scripts/tests/testUnit.sh",
     "test": "yarn lint && yarn test-unit && yarn test-integration",
-    "tdd:watch": "sh scripts/tests/tdd.sh",
```
Comment (pallabez) [inline package.json:14]: Can we not remove this line? I believe this is not related to swagger.

Comment (poonammohata) [reply package.json:14]: in this, just trailing comma is removed, not the whole line, getting error because of trailing comma

File: package.json [hunk 2]
```diff
@@ -11,8 +11,7 @@
...
-    "generate-api-schema": "node utils/generateAPISchema.js"
+    "tdd:watch": "sh scripts/tests/tdd.sh"
```
Comment (poonammohata) [inline package.json:14]: @pallabez  see it is here

Comment (pallabez) [reply package.json:14]: Oh. My Bad.

**Review by pallabez — APPROVED** [review]
LGTM :+1:
